test/hotspot/jtreg/compiler/loopopts/superword/RedTest_long.java

```diff
@@ -157,6 +142,17 @@ public static void reductionInit2(
     }
 
 
+    /* Require avx2 due to filtering out small vector lengths
+       for long and int (SuperWord::implemented). */
+    @Test
+    @IR(applyIfOr = {"SuperWordReductions", "false", "LoopMaxUnroll", "< 8"},
+        failOn = {IRNode.ADD_REDUCTION_VL})
+    @IR(applyIfCPUFeature = {"avx2", "false"},
```

Comment:
I'm afraid the restriction here is not safe because all non-avx2 machines would meet the condition, like aarch64 and riscv64. SVE supports `AndReductionV`, `OrReductionV `, `XorReductionV`, `AddReductionVL`, `AddReductionVD`, `SqrtVD`, and so on.